Dockerfile.assisted-service-debug

```diff
@@ -0,0 +1,10 @@
+ARG SERVICE=quay.io/ocpmetal/assisted-service:latest
+FROM registry.ci.openshift.org/openshift/release:golang-1.15 AS build-env
```

Comment:
`s/build-env/delve_download/` or `s/build-env/delve`, `build-env` is a bit misleading because you're not building anything, just downloading a delve binary